### PR TITLE
EG-1043, multi lingual Labels

### PR DESCRIPTION
**Pull request by cmrhmrc**

### CODE REVIEW

File: test/models/ocelot/LabelSpec.scala [hunk 1]
```diff
@@ -18,31 +18,45 @@ package models.ocelot
...
+  val labelWithSingleValue = """{"name":"BLAH","english":"39.99"}"""
+  val labelWithValues = """{"name":"BLAH","english":"Hello","welsh":"Welsh, Hello"}"""
 
   "Label" must {
     "deserialise " in {
       Json.parse(label).as[Label] shouldBe Label("BLAH")
-      Json.parse(labelWithValue).as[Label] shouldBe Label("BLAH", Some("39.99"))
-      Json.parse(labelWithType).as[Label] shouldBe Label("BLAH", None, Some(Currency))
-      Json.parse(labelWithValueAndType).as[Label] shouldBe Label("BLAH", Some("32.99"), Some(Currency))
+      Json.parse(labelWithSingleValue).as[Label] shouldBe Label("BLAH", Some("39.99"))
```
Comment (woollett-r) [inline test/models/ocelot/LabelSpec.scala:31]: Should we have a test for "labelWithValues" as well?

File: test/models/ocelot/LabelSpec.scala [hunk 2]
```diff
@@ -18,31 +18,45 @@ package models.ocelot
...
     }
 
+    "serialise from Label with both english and welsh values to json" in {
+      val dLabelWithValues: Label = Label("BLAH", Some("Hello"), Some("Welsh, Hello"))
+      Json.toJson(dLabelWithValues).toString shouldBe labelWithValues
+    }
+
+  }
+
+  "Label" must {
```
Comment (woollett-r) [inline test/models/ocelot/LabelSpec.scala:52]: You have repeated the test name. I would combine the tests or create a new name if that is more appropriate.

File: test/models/ocelot/LabelSpec.scala [hunk 3]
```diff
@@ -18,31 +18,45 @@ package models.ocelot
...
+    "contruct Value labels when presented with a single value" in {
```
Comment (woollett-r) [inline test/models/ocelot/LabelSpec.scala:53]: Am I missing something here? We seem to be testing that two identical statements give the same result.

File: test/models/ocelot/LabelSpec.scala [hunk 4]
```diff
@@ -90,25 +104,25 @@ class LabelSpec extends BaseSpec with ProcessJson {
     }
 
     "Return a map of new and updated labels on request" in {
-      val labelsMap = Map("X"->Label("X", Some("33.5")), "Y"->Label("Y", Some("4")), "Name" -> Label("Name", Some("Coltrane")))
+      val labelsMap = Map("X"->Label("X", Some("33.5")), "Y"->Label("Y", Some("4")), "Name" ->Label("Name", Some("Coltrane")))
```
Comment (woollett-r) [inline test/models/ocelot/LabelSpec.scala:107]: Nice

File: test/models/ocelot/LabelSpec.scala (hunk 2 above)
Comment (cmrhmrc) [reply test/models/ocelot/LabelSpec.scala:52]: will fix

File: test/models/ocelot/LabelSpec.scala (hunk 3 above)
Comment (cmrhmrc) [reply test/models/ocelot/LabelSpec.scala:53]: will fix

File: test/models/ocelot/LabelSpec.scala (hunk 4 above)
Comment (cmrhmrc) [reply test/models/ocelot/LabelSpec.scala:111]: 👍

**Pull request merged**